### PR TITLE
Fix images links

### PR DESCRIPTION
**Pull request by jamesetouma**
Fixed image links for Python_Tutorials

### CODE REVIEW

**Comment by oskooi** [comment]
Could you explain why this change is necessary?

**Comment by jamesetouma** [comment]
The images were not rendered for me on Firefox or Brave. Do they render for you if you view the markdown files in the Python_Tutorial folder? I believe it has to do with <center>...</center>. I'm not sure if gitlab changed their markdown API.

**Comment by oskooi** [comment]
I think this is a necessary change as well because [`<center>`](https://www.w3schools.com/tags/tag_center.asp) is an HTML tag used for aligning text and not images. (It was set up this way by @HomerReid when he created the first version of the documentation using markdown.) Using `<center>` for images works on Chrome and the reason it has not been an issue is because we have not really tested it on other browsers until now.

File: doc/docs/Python_Tutorials/Basics.md [hunk 1]
```diff
@@ -51,7 +51,9 @@ geometry = [mp.Block(mp.Vector3(mp.inf,1,mp.inf),
 
 The waveguide is specified by a `Block` (parallelepiped) of size $\infty \times 1 \times \infty$, with $ε=12$, centered at (0,0) which is the center of the cell. By default, any place where there are no objects there is air ($ε=1$), although this can be changed by setting the `default_material` variable. The resulting structure is shown below.
 
-<center>![](../images/Python-Tutorial-wvg-straight-eps-000000.00.png)</center>
+<p align="center">
+  <img  src="../images/Python-Tutorial-wvg-straight-eps-000000.00.png">
```
Comment (oskooi) [inline doc/docs/Python_Tutorials/Basics.md:55]: Remove the extra space between `img` and `src`. This also applies to several other instances in this PR.

File: doc/docs/Python_Tutorials/Basics.md [hunk 2]
```diff
@@ -1115,12 +1126,12 @@ unix% convert ring-ez-*.png ring-ez-0.118.gif
 ```
 
 The resulting animations for (from left to right) 0.118, 0.147, and 0.175, are below, in which you can clearly see the radiating fields that produce the losses:
+<p align="center">
+  <img src="../images/Tut-ring-ez-0.118.gif">
```
Comment (oskooi) [inline doc/docs/Python_Tutorials/Basics.md:1130]: Remove extra blank space to correct alignment.

File: doc/docs/Python_Tutorials/Resonant_Modes_and_Transmission_in_a_Waveguide_Cavity.md [hunk 3]
```diff
@@ -408,10 +408,18 @@ It is usually a good idea to examine the field patterns for any modes that you a
 + $k_x=0.1$, $ω=0.4811-0.0017i$ leaky mode
 + $k_x=0.3$, $ω=0.8838-0.0018i$ leaky mode
 + $k_x=0.25$, $ω=0.2506$ light-cone (extended) mode
+<p align="center">
+  <img  src="../images/Holey-wvg-kx=0.4-w=0.1896.gif">
+
```
Comment (oskooi) [inline doc/docs/Python_Tutorials/Resonant_Modes_and_Transmission_in_a_Waveguide_Cavity.md:413]: Remove blank lines between `img` lines.

**Comment by jamesetouma** [comment]
changes have been made

File: doc/docs/Python_Tutorials/Cylindrical_Coordinates.md [hunk 4]
```diff
@@ -163,11 +167,9 @@ For a given mode of the ring resonator, it is often useful to know how sensitive
 
 [Pertubation theory for Maxwell equations involving high index-contrast dielectric interfaces](http://math.mit.edu/~stevenj/papers/KottkeFa08.pdf) is reviewed in Chapter 2 of [Photonics Crystals: Molding the Flow of Light, 2nd Edition (2008)](http://ab-initio.mit.edu/book/). The formula (equation 30 on p.19) for the frequency shift $\Delta \omega$ resulting from the displacement of a block of $\varepsilon_1$-material towards $\varepsilon_2$-material by a distance $\Delta h$ (perpendicular to the boundary) is:
 
-<center>
```
Comment (oskooi) [inline doc/docs/Python_Tutorials/Cylindrical_Coordinates.md:166]: Is it possible to also apply the `<p align="center">...</p>" HTML construct to center Latex equations?

Comment (jamesetouma) [reply doc/docs/Python_Tutorials/Cylindrical_Coordinates.md:166]: I'll give it a try later today.

Comment (jamesetouma) [reply doc/docs/Python_Tutorials/Cylindrical_Coordinates.md:166]: These three gave the same display. 
The \$\$ ... \$\$ is supposed to center it, at least from the pure $\LaTeX$ behavior. 


<center>

$$ \sigma_{scatt} = \int_0^{2\pi} d\phi \int_0^{\pi} \sigma_{diff}(\phi,\theta)\sin(\theta)d\theta $$

</center>

$$ \sigma_{scatt} = \int_0^{2\pi} d\phi \int_0^{\pi} \sigma_{diff}(\phi,\theta)\sin(\theta)d\theta $$

<p align="center">
   $$ \sigma_{scatt} = \int_0^{2\pi} d\phi \int_0^{\pi} \sigma_{diff}(\phi,\theta)\sin(\theta)d\theta $$
</p>

File: doc/docs/Python_Tutorials/Optical_Forces.md [hunk 5]
```diff
@@ -3,10 +3,10 @@
...
+<p align="center">
+  <img src="../images/Waveguide_forces.png">
+    </p>
```
Comment (oskooi) [inline doc/docs/Python_Tutorials/Optical_Forces.md:8]: `</p>` is indented incorrectly.

Comment (jamesetouma) [reply doc/docs/Python_Tutorials/Optical_Forces.md:8]: I'll go through one more time, fix the indentations, and remove the "center" from the equations (and just keep \$\$ \... \$\$)

**Comment by oskooi** [comment]
Note that it is not necessary to create a separate commit for each change to each file. You can (and in this case should) combine all the changes within a single commit.

**Comment by jamesetouma** [comment]
I was editing in the browser. I'll keep that in mind in the future. Maybe you can squash those commits if these edits get accepted.

**Review by oskooi — APPROVED** [review]
LGTM.

**Comment by stevengj** [comment]
Thanks!

**Comment by oskooi** [comment]
Unfortunately, this PR has removed all the images from the online user manual, e.g., see:
https://meep.readthedocs.io/en/latest/Python_Tutorials/Basics/

**Comment by jamesetouma** [comment]
Yes I saw that. The html tags maintain their relative path; ie ../images/x.png, while the site folder needs ../../images/x.png to render correctly. I am not sure why this is the case. However, if we use plain markdown tags (  ), they both render correctly, however, the images will not be centered.

**Comment by jamesetouma** [comment]
I submitted another PR -  plz see comment in the request.

**Comment by oskooi** [comment]
One solution to centering images in markdown *without* adding HTML tags is via CSS: https://msudol.com/how-to-align-images-in-markdown-without-html-or-extensions/. We can do something like:
```
![<>](../image.png)
```
(i.e., adding the `<>` inside the square brackets.)

Perhaps we can test this feature on just a single image first via a PR and if it works apply it to all the other images (via another PR).

**Comment by jamesetouma** [comment]
Sure!

**Comment by jamesetouma** [comment]
Just updated my PR branch. 

css/extra.css is updated with the two methods
Python_tutorials/Basics.md: updated the last image in the file using both methods. Please let me know which one you prefer. 

I noticed that the MD file did not change but the HTML file did. Maybe you want to check it on your side.

File: doc/docs/Python_Tutorials/Local_Density_of_States.md [hunk 6]
```diff
@@ -222,7 +223,7 @@ if __name__ == '__main__':
...
+We consider the simple example of a 2D perfect-metal $a$x$a$ cavity of finite thickness 0.1$a$, with a small notch of width $w$ on one side that allows the modes to escape. The nice thing about this example is that in the absence of the notch, the lowest-frequency $E_z$-polarized mode is known analytically to be $E_z^{(1)}=\frac{4}{a^2}\sin(\pi x/a)\sin(\pi \gamma/a)$, with a frequency $\omega^{(1)}=\sqrt{2}\pi c/a$ and modal volume $V^{(1)}=a^2/4$. The notch slightly perturbs this solution, but more importantly the opening allows the confined mode to radiate out into the surrounding air, yielding a finite $Q$. For $w \ll a$, this radiative escape occurs via an evanescent (sub-cutoff) mode of the channel waveguide formed by the notch, and it follows from inspection of the evanescent decay rate $\sqrt{(\pi/\omega)^2-(\omega^{(1)})^2}/c$ that the lifetime scales asymptotically as $Q^{(1)} \sim e^{\\#/\omega}$ for some coefficient \#.
```
Comment (oskooi) [inline doc/docs/Python_Tutorials/Local_Density_of_States.md:226]: Putting *two* slashes before the `#` character in the equation on the last line of this paragraph is causing it not to render in the online user manual: https://meep.readthedocs.io/en/latest/Python_Tutorials/Local_Density_of_States/ .

Comment (jamesetouma) [reply doc/docs/Python_Tutorials/Local_Density_of_States.md:226]: Typo - I'll fix

**Pull request merged**